src/aap_eda/logging/startup.py

```diff
@@ -0,0 +1,94 @@
+#  Copyright 2025 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import logging
+import platform
+
+from django.conf import settings
+
+from aap_eda.logging import UnconditionalLogger
```

Comment:
@Alex-Izquierdo Do we need another logger to be defined, can't we just explicitly set the log level in this function. for example
```
from django.apps import AppConfig
import logging
from aap_eda.utils import get_eda_version
logger = logging.getLogger(__name__)


class CoreConfig(AppConfig):
    default_auto_field = "django.db.models.BigAutoField"
    name = "aap_eda.core"

    def ready(self):
        logger.setLevel(logging.INFO)
        logger.info(f"EDA Version: {get_eda_version()}")
        # make sure we apply DAB decorations in case they are not yet imported
        from aap_eda.api.views import dab_decorate  # noqa: F401

```

Reply:
Hi @mkanoor I'm not adding a new logger, it is just a simple wrapper that enforces the message get printed regardless of the log level or logger. Here we have a simple situation: we want to print something at startup regardless of the logger and regardless of the log level. I don't think we should be using a level that doesn't match the app config and we should not be taking care of changing the level for an existing logger at runtime. 

I would agree that creating a new logger with a new custom level is probably excessive for what we need to do, so to me simplest solution is this.

Reply:
I have removed the STARTUP_LOGGING_ENABLED setting as we discussed in the chat. Please review again.